TravelerApp/TravelerConsole/Program.cs

```diff
@@ -51,7 +58,16 @@ static void Main(string[] args)
 
             int numberOfRecords = 10;
             DrawTable.Hotelinfo(dataReaded, numberOfRecords);
-
+            //Zmieniam funkcję wyszukiwanie po lokalizacji na typ delegata ja nazwałam go Del, żebym mogła dodać ją do parametrów funkcji
+            Del jakas = TravelerAppCore.Models.Search.byLocalisation;
+           
+           
+            List<Item> lista = new List<Item>();//lista znaków jakie może wpisywać użytkownik, żeby poruszać się po menu
```

Comment:
ogólnie to niech zostanie tak jak jest teraz ale kod powinien być tak napisany że jak go czytamy to nie potrzebujemy dodatkowych komentarzy które nam objaśniają co on robi :)

Reply:
Ok to może jednak usunę.